src/utils.rs

```diff
@@ -33,6 +33,20 @@ pub async fn create_response_text(
     create_embed_response(http, interaction, embed).await
 }
 
+pub async fn create_error_text(
```

Comment:
Can we just call it "create_followup_message" as this can be used for other uses in the future and not only errors

Reply:
Sure, do you want me go ahead and implement the equivalent functions for create_response, edit_response, edit_response_text, edit_response_text, and edit_embed_response as well or just what we need for now?

Reply:
Just what we need for now and yeah I agree